CHANGELOG.md

```diff
@@ -1,67 +1,58 @@
-## [1.2.4](https://github.com/Asymmetrik/graphql-fhir/compare/1.2.3...1.2.4) (2019-08-20)
-
+## [1.3.1](https://github.com/Asymmetrik/graphql-fhir/compare/1.2.4...1.3.1) (2021-09-17)
 
 ### Bug Fixes
 
-* updated package containing vulnerability ([0d99acd](https://github.com/Asymmetrik/graphql-fhir/commit/0d99acd))
-* upgrade packages ([#30](https://github.com/Asymmetrik/graphql-fhir/issues/30)) ([5cd0619](https://github.com/Asymmetrik/graphql-fhir/commit/5cd0619))
+- update deps ([35a2e9a](https://github.com/Asymmetrik/graphql-fhir/commit/35a2e9ad3a9d91e96238e20f40dc26428864a635))
 
+## [1.3.0](https://github.com/Asymmetrik/graphql-fhir/compare/1.3.0...1.2.4) (2021-09-15)
```

Comment:
# clean up
Either use 1.2.4 to 1.3.1 and delete 1.2.4 to 1.3.0, or use 1.2.4 to 1.3.0 and 1.3.0 to 1.3.1